templates/zerver/help/enable-emoticon-translations.md

```diff
@@ -0,0 +1,24 @@
+# Enable emoticon translation
+
+If you use emoticons like `:)` or `:/`, you can have them translated into
+emoji equivalents like 😃 or 😕 automatically by Zulip.
```

Comment:
Look at how our emoji are done in these docs; there might be a way to have this be an image.

Reply:
@timabbott OK, thanks! I changed it so that it uses an image now, like the way `add-emoji` does it. I also had to convert the Markdown table to HTML because I was getting linter errors.